src/components/switch/switch.component.tsx

```diff
@@ -117,7 +119,7 @@ const Switch: React.FC<SwitchProps> = (incomingProps) => {
               inputRange: [0, 1],
               outputRange: [
                 getThemeColor(theme.colors, bg),
-                getThemeColor(theme.colors, `${props.colorScheme}.500`),
+                getThemeColor(theme.colors, props.colorScheme),
```

Comment:
`getThemeColor` looks to wait a color tint (`color.tint` syntax) as second argument, not a `colorScheme` that the name of a set of tints (`color` syntax).